packages/block-editor/src/components/rich-text/test/index.perf-test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { measurePerformance } from 'reassure';
+import { fireEvent } from '@testing-library/react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+describe( 'RichText Performance', () => {
+	it( 'measures performance', async () => {
```

Comment:
The current description is a bit repetitive and uninformative. What do you think about the following approach for this and the other tests? 
```suggestion
describe( 'RichText', () => {
	it( 'performance is stable', async () => {
```

Reply:
Updated.